Data/Primitive/Array.hs

```diff
@@ -1,6 +1,9 @@
 {-# LANGUAGE CPP, MagicHash, UnboxedTuples, DeriveDataTypeable, BangPatterns #-}
 {-# LANGUAGE RankNTypes #-}
 {-# LANGUAGE TypeFamilies #-}
+#if __GLASGOW_HASKELL__ >= 708
```

Comment:
Shouldn't this be 706? (That's the first GHC version that reliably supported `PolyKinds`.)

Reply:
I mistakenly though it didn't stabilize till later. Fixed!